### PR TITLE
Reduce IntelliJ warnings

### PR DESCRIPTION
**Pull request by brfrn169**
The PR reduces the number of the IntelliJ warnings. Please take a look!

### CODE REVIEW

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

File: core/src/main/java/com/scalar/db/storage/jdbc/RdbEnginePostgresql.java [hunk 1]
```diff
@@ -184,12 +184,10 @@ public String getDataTypeForEngine(DataType scalarDbDataType) {
 
   @Override
   public String getDataTypeForKey(DataType dataType) {
-    switch (dataType) {
-      case TEXT:
-        return "VARCHAR(10485760)";
-      default:
-        return null;
+    if (dataType == DataType.TEXT) {
```
Comment (komamitsu) [inline core/src/main/java/com/scalar/db/storage/jdbc/RdbEnginePostgresql.java:187]: Honestly speaking I prefer the original one in terms of extensibility 😆

**Review by komamitsu — APPROVED** [review]
LGTM! Thank you!

File: core/src/main/java/com/scalar/db/storage/jdbc/RdbEnginePostgresql.java (hunk 1 above)
Comment (brfrn169) [reply core/src/main/java/com/scalar/db/storage/jdbc/RdbEnginePostgresql.java:187]: My IntelliJ complains the original one for some reason...

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**